tests/bootstrap.php

```diff
@@ -1,14 +1,10 @@
 <?php declare(strict_types=1);
 
-require_once __DIR__ . '/../vendor/autoload.php';
+use Neusta\Pimcore\TestingFramework\Pimcore\BootstrapPimcore;
 
-$setEnv = function (string $name, string $value): void {
-    putenv("{$name}=" . $_ENV[$name] = $_SERVER[$name] = $value);
-};
+include __DIR__ . '/../vendor/autoload.php';
```

Comment:
I think we should stick with `require_once` because we really *need* this file.

```suggestion
require_once __DIR__ . '/../vendor/autoload.php';
```

Reply:
added in [refactor] suggested changes